src/components/Navbar.tsx

```diff
@@ -0,0 +1,41 @@
+function Navbar() {
+// This is just a dummy test for navbar. TODO Implement me later
+    return ( 
+        <nav className="navbar navbar-expand-lg bg-body-tertiary">
+            <div className="container-fluid">
+                <a className="navbar-brand" >Navbar</a>
+                <button className="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span className="navbar-toggler-icon"></span>
+                </button>
+                <div className="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul className="navbar-nav me-auto mb-2 mb-lg-0">
+                    <li className="nav-item">
+                    <a className="nav-link active" aria-current="page" >Home</a>

```

Comment:
Indentation seems a bit off in the entire file (and in other files as well) , for example here I think the <a> tag should have one more TAB. My recommendation is to download the [Prettier](https://prettier.io/) extension for VS Code and configure it to format your code every time you Ctrl + S. This way you will get industry standard code formatting without ever having to think of it again.